src/maps/map.py

```diff
@@ -55,7 +59,7 @@ def add_areas(self, dimension: dict, reports: Reports, show=False, scale=None):
             colourmap = colourmap.to_step(data=non_zero_score_col, quantiles=[0, 0.2, 0.4, 0.6, 0.8, 1])
         else:
             colourmap = branca.colormap.LinearColormap(colors=["#4dac26", "#b8e186", "#f1b6da", "#d01c8b"], index=scale["index"], vmin=scale["min"], vmax=scale["max"],).to_step(
-                index=scale["boundaries"]
+                index=scale["index"]
             )
```

Comment:
is changing from boundaries to index intentional? map_uptake_script's static scale does use the boundaries option - not sure I ever looked into this so could just be removed

Reply:
ah so would help if I understood my code in the first place ... I think I've got it now, it should be 'boundaries' and not 'index'. The reason is as follows:

'index' sets the value that relates to each colour - and must be the same length as the 'colors' argument of the LinearColormap. Say red is 0, white is 0.5 and blue is 1. This generates a continuous colour spectrum (LinearColormap). By definition 'index' must only have four values.

'boundaries' allows you to split up the continuous colour spectrum (LinearColormap) into a variable number of discrete colours (which are easier to distinguish on a map). This can be a list of any length.

Apologies should have documented this better, it even managed to confuse me. I'm not totally convinced I like using the magic 'scale' dictionary ... but for now should be the 'boundary' key. Feel free to fix if you get chance before me.